src/Equinox/Core.fs

```diff
@@ -34,7 +34,7 @@ and [<Struct; NoEquality; NoComparison>] StreamToken = { value: obj; version: in
 type internal Impl() =
 
     static let run (stream: IStream<'e, 's>)
-            (decide: Func<struct (_ * _), CancellationToken, Task<struct ('r * 'e seq)>>)
+            (decide: Func<struct (_ * _), CancellationToken, Task<struct ('r * 'e[])>>)
```

Comment:
L43 Array.ofseq is now redundant (and can become a match!)

Reply:
Done